diff --git a/globals.d.ts b/globals.d.ts
index ab3685cfff123457f3262965c300c11d477df301..07839fff7e0f48efeb104fa6d0a051517e4aa561 100644
--- a/globals.d.ts
+++ b/globals.d.ts
@@ -1556,6 +1556,15 @@ declare var URL: Bun.__internal.UseLibDomIfAvailable<
   }
 >;
 
+/**
+ * The **`AbortController`** interface represents a controller object that allows you to abort one or more Web requests as and when desired.
+ *
+ * [MDN Reference](https://developer.mozilla.org/docs/Web/API/AbortController)
+ */
+interface AbortController {
+  readonly signal: AbortSignal;
+  abort(reason?: any): void;
+}
 declare var AbortController: Bun.__internal.UseLibDomIfAvailable<
   "AbortController",
   {
diff --git a/index.d.ts b/index.d.ts
index a5430eeec9532b109b37a875eb43ec6d7089c6e4..b8575a0b2eb94a3b40056faceba876fe3d40d38b 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -26,6 +26,6 @@
 
 /// <reference path="./bun.ns.d.ts" />
 
-// @ts-ignore Must disable this so it doesn't conflict with the DOM onmessage type, but still
+// Must disable this so it doesn't conflict with the DOM onmessage type, but still
 // allows us to declare our own globals that Node's types can "see" and not conflict with
-declare var onmessage: never;
+declare var onmessage: Bun.__internal.UseLibDomIfAvailable<"onmessage", never>;
